### ISSUE TITLE
Template: Links not working at 400% zoom. - 1.4.10 Reflow 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#84]
When the page is zoomed in to 400%, the buttons across the top of the page are removed and are not accessible to users "A-Z
Contact
Directories
Locus
Support LUC"

ADDITIONALLY, there is an arrow going into some content that shows third level pages. Since this is not functional on the normal desktop view, this is a failure as there is different content at different viewpoints

**Comment by syarwick23** [comment]
Issue fixed with new template
